### ISSUE TITLE
"regeneratorRuntime is not defined"

### ISSUE THREAD

**Issue by leebenson** [chentsulin/koa-graphql#57]
with 5.3, I now get this:

```
general error:  ReferenceError: regeneratorRuntime is not defined
    at graphqlHTTP (/Users/dev/app/node_modules/koa-graphql/dist/index.js:41:10)

```

**Comment by leebenson** [comment]
actually, ignore - this is a local issue

**Comment by leebenson** [comment]
re-opening -- seems to be a regression from 0.5.2+ (works fine in 0.5.1)

**Comment by chentsulin** [comment]
Oh, it should be introduced by babel 6. You can use `babel-polyfill` for a quick fix.

And we should fix it via `babel-runtime`

```
$ npm install --save-dev babel-plugin-transform-runtime
$ npm install --save babel-runtime
```

```js
{
  "plugins": [
    ["transform-runtime", {
      "polyfill": false,
      "regenerator": true
    }]
  ]
}
```

**Comment by chentsulin** [comment]
@leebenson Released as v0.5.4: https://github.com/chentsulin/koa-graphql/releases/tag/v0.5.4
